packages/itwinui-variables/src/colors.scss

```diff
@@ -28,12 +28,12 @@
 
   @if $contrast == default {
     $background-statuses: (
-      background-primary: hsl(203 100% 44%),
-      background-positive: hsl(95 72% 37%),
-      background-warning: hsl(33 89% 51%),
+      background-primary: hsl(203 100% 39%),
+      background-positive: hsl(95 72% 30%),
+      background-warning: hsl(33 89% 36%),
       background-negative: hsl(0 91% 43%),
-      background-primary-hover: hsl(203 100% 34%),
-      background-positive-hover: hsl(95 72% 27%),
+      background-primary-hover: hsl(203 100% 29%),
+      background-positive-hover: hsl(95 72% 20%),
```

Comment:
@mayank99 these might need to be moved because the hover color changes between light & dark themes.

Main branch shows in light theme the hover gets darker / dark theme the hover gets lighter:
![Screen Shot 2022-08-12 at 9 51 01 AM](https://user-images.githubusercontent.com/849817/184367463-46a97de4-279e-41a8-8f2e-88992337116e.png)
![Screen Shot 2022-08-12 at 9 50 41 AM](https://user-images.githubusercontent.com/849817/184367533-fb165085-0b5f-4c51-8f72-5fe9b56224d2.png)

This branch both themes hover gets darker:
![Screen Shot 2022-08-12 at 9 53 01 AM](https://user-images.githubusercontent.com/849817/184367830-b17008b0-1f05-4d08-a7fb-cdd04f604c09.png)
![Screen Shot 2022-08-12 at 9 53 27 AM](https://user-images.githubusercontent.com/849817/184367903-fb105944-79e7-424b-a2d8-2e25a6910e3a.png)

I'd like for dark theme's hover to get lighter.